app/backend/wells/models.py

```diff
@@ -1309,3 +1312,97 @@ def __str__(self):
                                                          self.screen_to)
         else:
             return 'well {} {} {}'.format(self.well, self.screen_from, self.screen_to)
+
+
+class AquiferVulnerablityCode(AuditModel):
+    """
+    Demand choices for describing Aquifer 
+    -------------------
+    High
+    Low
+    Moderate
+
+    """
+    code = models.CharField(primary_key=True, max_length=1, db_column='aquifer_vulnerablity_code')
+    description = models.CharField(max_length=100)
+    display_order = models.PositiveIntegerField()
+
+    effective_date = models.DateTimeField(blank=True, null=True)
+    expiry_date = models.DateTimeField(blank=True, null=True)
+
+    class Meta:
+        db_table = 'aquifer_vulnerablity_code'
+        ordering = ['display_order', 'code']
+        verbose_name_plural = 'Aquifer Vulnerability Codes'
+
+    def __str__(self):
+        return '%s - %s' % (
+            self.code,
+            self.description
+        )
+
+
+"""
+    Hydraulic properties of the well, usually determined via tests.
+"""
+class HydraulicProperty(AuditModel):
+
+    hydraulic_property_guid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    well_tag_number = models.ForeignKey(Well, db_column='well_tag_number', to_field='well_tag_number',
```

Comment:
Can this field rather be called "well"? Reason being, in django, this column maps to the well object, not the tag number.

Reply:
yes.. fixed but note this same file has another usage that needs to be fixed:
"class AquiferWell"